dash_table_base/__init__.py

```diff
@@ -18,6 +15,9 @@
     )
     _sys.exit(1)
 
+from ._imports_ import *  # noqa: E402, F401, F403
+from ._imports_ import __all__  # noqa: E402
```

Comment:
The test above needs to happen before these imports or it's useless, because the components will try to use `dash.development`.